libflowlimit/RPCQPSLimiter.h

```diff
@@ -42,6 +42,7 @@ class RPCQPSLimiter
     RateLimiter::Ptr getQPSLimiterByGroupId(dev::GROUP_ID const& _groupId);
 
     virtual bool acquire(int64_t const& _requiredPermits = 1);
+    virtual void acquireWithoutWait(int64_t const& _requiredPermits = 1);

```

Comment:
int64_t in unnecessary reference
```suggestion
    virtual void acquireWithoutWait(int64_t  _requiredPermits = 1);

```

Reply:
done.